<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1613</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MEMBER PROFILE</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>760</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1364</width>
      <height>758</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="national_id">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>270</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>550</y>
       <width>621</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>OTHER INFORMATION   </string>
     </property>
     <widget class="QLabel" name="attendance">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>LM Roman 17</family>
       </font>
      </property>
      <property name="text">
       <string>MEMBER ATTENDANCE: 14 OUT OF 30 MEETINGS</string>
      </property>
     </widget>
     <widget class="QLabel" name="events">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>LM Roman 17</family>
       </font>
      </property>
      <property name="text">
       <string>NUMBERR OF EVENTS: NONE</string>
      </property>
     </widget>
    </widget>
    <widget class="QDateEdit" name="date_joined">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>170</y>
       <width>441</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>320</y>
       <width>621</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>FAMILY AND EMERGENCY CONTACTS INFORMATION    </string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>601</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout"/>
     </widget>
    </widget>
    <widget class="QLabel" name="profilePicLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../media/default_pic.png</pixmap>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>711</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>CONTACT INFORMATION    </string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>LM Roman 17</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ADDRESS</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>LM Roman 17</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EMAIL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="address">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>phone number 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>LM Roman 17</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PHONE NUMBER(s)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>phone number 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LM Roman 17</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DATE JOINED</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>711</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>FINANCIAL  INFORMATION    </string>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>691</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="tt_lb1">
         <property name="text">
          <string>TOTAL CONTRIBUTIONS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="total_contribs"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ttlb2">
         <property name="text">
          <string>TOTAL PENDING PAYMENTS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="total_loans"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ttlb3">
         <property name="text">
          <string>TOTAL ARREARS AND PENALTIES</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="total_arrears"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLineEdit" name="full_name_box">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>106</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string>Raymond Reddington</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>260</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LM Roman 17</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NATIONAL ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LM Roman 17</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FULL NAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>TeX Gyre Termes Math</family>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Raymond Reddington's Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(98, 160, 234);</string>
     </property>
     <property name="text">
      <string>save changes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LM Roman 17</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>STATUS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="active_check_box">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>220</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSerif</family>
       <pointsize>18</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>ACTIVE</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_btn">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
